pipeline/pipeline.go

```diff
@@ -38,6 +39,8 @@ const (
 	antispamUnbanIterations = 4
 	metricsGenInterval      = time.Hour
 
+	workEventsGauge         = "work_events_gauge"
```

Comment:
https://prometheus.io/docs/practices/naming/, don't add metric type into metric name

Reply:
better report pool size and events in pool rather than work/wait event count